ntoskrnl/include/internal/io.h

```diff
@@ -629,7 +629,7 @@ IopFreeDeviceNode(
 
 NTSTATUS
 NTAPI
-IopQueryDeviceCapabilities(PDEVICE_NODE DeviceNode,
+PiSetDeviceCapabilities(PDEVICE_NODE DeviceNode,
                            PDEVICE_CAPABILITIES DeviceCaps);
```

Comment:
```suggestion
    PDEVICE_CAPABILITIES DeviceCaps);
```

Reply:
This suggestion doesn't make much more sense.

Reply:
It does. Because it never ever has to be changed again, no matter to which name the function would be renamed to.

Reply:
I admit the indentation in my commit was not the best. But the suggestion to drop the indentation for the second parameter completely looks odd since all other function declarations use it.

Reply:
Well, this change introduces an odd indentation for the function arguments, as you can see:

```c
PiSetDeviceCapabilities(PDEVICE_NODE DeviceNode,
                           PDEVICE_CAPABILITIES DeviceCaps);
```

The first argument is located more to the left, and the second is more to the right. In order to make this consistent, I would suggest doing it like in other places:

```c
PiSetDeviceCapabilities(
    PDEVICE_NODE DeviceNode,
    PDEVICE_CAPABILITIES DeviceCaps);
```

And ideally add SAL2 annotation here as well.